### ISSUE TITLE
accessibility: add hidden tale view of WTFcsv histogram data

### ISSUE THREAD

**Issue by rahulbot** [c4fcm/DataBasic#96]
> In the previous paragraph, I showed some run-together text which I assumed represented a word cloud. However, I'm seeing it in many places in the report generated by WTFCSV. Each entry labeled counts # of rows:
...
Age
Counts# of rows2422181928010203040
...
SibSp
Counts# of rows012430200400600800
...
Parch
Counts# of rows012350250500750
...
Ticket
Categories# of rowsCA_ 234316013470823101295CA 214402.557.5
...
Survived
Counts# of rows
...
Name
mrmissmrswilliamjohnmasterhenryjamesgeorgecharlesthomasmaryedwardannajosephelizabethfrederickjohanrichardsamuelarthuralfredmargaretmariaalexanderpeterjrroberternestleonardanderssonkarlalbertaliceanniemartinvictorhjfrankhelensagededrkatecatherinedavidedithmariejohnson
...

A look at the markup suggests that these items are the screen reader's lame interpretation of SVG.  We suggest using aria-hidden to hide the svg, and then providing a separate html table showing the same results. You can use css to move the html table off-screen (either moving it far off-left or above top, or clip to zero size). This hides from sighted users, but since it is in the dom and does not have css display:none or visibility:hidden (which hide from screen reader users as well as sighted users), screen reader users will see it.

**Comment by rahulbot** [comment]
Added tables for all types of cards
